storage/database/dynamodb_test.go

```diff
@@ -91,6 +91,28 @@ func (s *SuiteDynamoDB) TestDynamoDB_Put() {
 	s.NoError(returnedErr)
 }
 
+func (s *SuiteDynamoDB) TestDynamoDB_Put_EmptyVal() {
+	dynamo, err := newDynamoDB(GetTestDynamoConfig())
+	if err != nil {
+		s.FailNow("failed to create dynamoDB", err)
+	}
+	s.dynamoDBs = append(s.dynamoDBs, dynamo)
+
+	// write nil value
+	key := common.MakeRandomBytes(32)
+	s.Nil(dynamo.Put(key, nil))
+
+	// get nil value
+	ret, err := dynamo.Get(key)
+	s.Equal([]byte{}, ret)
```

Comment:
Shouldn't `ret` be `nil`?

Reply:
@miloveme LevelDB `Get()` returns  `[]byte{}` for `nil` value. I changed MemoryDB `Get()` to adapt the same convention. Also, I applied the test case reading/writing `nil` value to all database types.

Reply:
Thanks for your explanation!